.github/mergify.yml

```diff
@@ -4,24 +4,7 @@ defaults:
       assignees:
         - "{{ author }}"
 
-queue_rules:
-  - name: default
-    conditions:
-      - "#approved-reviews-by>=1"
-
 pull_request_rules:
-  - name: automerge to the base branch with label automerge and branch protection passing
-    conditions:
-      - "#approved-reviews-by>=1"
-      - base=osmo/v0.47.5
-      - label=A:automerge
-    actions:
-      queue:
-        name: default
-        method: squash
-        commit_message_template: |
-          {{ title }} (#{{ number }})
-          {{ body }}
```

Comment:
Note: VS code was complaining about these so I removed them because we don't need anyways